### DISCUSSION TITLE
read in the mm10 specific feather files

### DISCUSSION THREAD

**Discussion by sylestiel** [aertslab/SCENIC#276] [R] [answered]
@s-aibar 

Can anyone assist me with the scripts needed to read in the mm10 specific feather files? Something is amiss. Despite downloading the mm10 specific files to the dbDir cisTarget_databases the error reads as follows:

Error in initializeScenic(org = "mgi", dbDir = "cisTarget_databases", : The following RcisTarget databases were not found: - cisTarget_databases/mm9-500bp-upstream-7species.mc9nr.feather - cisTarget_databases/mm9-tss-centered-10kb-7species.mc9nr.feather Make sure the arguments 'dbDir' and 'dbs' are correct.

_Originally posted by @sylestiel in https://github.com/aertslab/SCENIC/issues/18#issuecomment-867945269_

**Comment by cytokine4242** [comment, marked as answer]
Changing the DefaultDbNames to the mm10 files worked for me.

```
defaultDbNames$mgi[1] <- "mm10__refseq-r80__10kb_up_and_down_tss.mc9nr.feather"
defaultDbNames$mgi[2] <- "mm10__refseq-r80__500bp_up_and_100bp_down_tss.mc9nr.feather"
defaultDbNames
scenicOptions <- initializeScenic(org="mgi", dbs = defaultDbNames[["mgi"]], dbDir="cisTarget_databases", nCores=4)

```

**Comment by sylestiel** [comment]
Hi cytokine4242,

I will definitely give it a try.

Thanks!
sylestiel
